includes/Core/Modules/Module.php

```diff
@@ -285,7 +285,8 @@ public function is_connected() {
 				continue;
 			}
 
-			if ( ! isset( $datapoint_services[ $dataset->datapoint ] ) ) {
+			$datapoint_name = "{$dataset->method}:{$dataset->datapoint}";
```

Comment:
I originally suggested to inline this since there only seems to be 2 instances, but if you'd prefer to keep it as a variable that's okay, however let's use a name like `$definition_key` to not confuse it with `$dataset->datapoint` which is more like the "name" of the datapoint (e.g. `accounts` or `properties-profiles`).

Reply:
Updated.